.github/workflows/package-publish.yml

```
@@ -0,0 +1,21 @@
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install Package
+        run: yarn
+      - name: Build
+        run: yarn clean && yarn build
+      - name: Publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npm whoami && npm publish --access public
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)